Upstream-stats: Inappropriate

Signed-off-by: Maksim Sisov <msisov@igalia.com>
---
From b1c1f13a173acee5be4ecfa991a6062c850655f5 Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Mon, 17 Sep 2018 13:07:59 +0300
Subject: [PATCH] Workaround gcc bug "protected within this context"

../../third_party/blink/renderer/core/streams/transform_stream.cc:31:23:
error:
'blink::TransformStream::FlushAlgorithm::FlushAlgorithm(blink::TransformStreamTransformer*,
blink::ScriptState*, blink::ExceptionState&)' is protected within this
context
     auto* algorithm = new T(transformer, script_state,
exception_state);
                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
---
 third_party/blink/renderer/core/streams/transform_stream.cc | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/third_party/blink/renderer/core/streams/transform_stream.cc b/third_party/blink/renderer/core/streams/transform_stream.cc
index a117c44cfcf2..77f6ee067629 100644
--- a/third_party/blink/renderer/core/streams/transform_stream.cc
+++ b/third_party/blink/renderer/core/streams/transform_stream.cc
@@ -37,15 +37,14 @@ class TransformStream::Algorithm : public ScriptFunction {
     ScriptFunction::Trace(visitor);
   }
 
- protected:
-  Algorithm(TransformStreamTransformer* transformer,
+ Algorithm(TransformStreamTransformer* transformer,
             ScriptState* script_state,
             ExceptionState& exception_state)
       : ScriptFunction(script_state),
         transformer_(transformer),
         interface_name_(exception_state.InterfaceName()),
         property_name_(exception_state.PropertyName()) {}
-
+ protected:
   // AlgorithmScope holds the stack-allocated objects used by the CallRaw()
   // methods for FlushAlgorithm and TransformAlgorithm.
   class AlgorithmScope {
@@ -83,7 +82,7 @@ class TransformStream::Algorithm : public ScriptFunction {
 };
 
 class TransformStream::FlushAlgorithm : public TransformStream::Algorithm {
- protected:
+ public:
   using Algorithm::Algorithm;
 
  private:
@@ -101,7 +100,7 @@ class TransformStream::FlushAlgorithm : public TransformStream::Algorithm {
 };
 
 class TransformStream::TransformAlgorithm : public TransformStream::Algorithm {
- protected:
+ public:
   using Algorithm::Algorithm;
 
  private:
-- 
2.11.0

